src/Qir/Runtime/stdlib/include/qir_stdlib.h

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <cstdint>
+
+#ifdef _WIN32
+#define QIR_SHARED_API __declspec(dllexport)
```

Comment:
I have an impression that one more condition is missing, totally something like
```cpp
#ifdef _WIN32
#ifdef EXPORT_QIR_API
#define QIR_SHARED_API __declspec(dllexport)
#else
#define QIR_SHARED_API __declspec(dllimport)
#endif
#else
#define QIR_SHARED_API
#endif
```

Reply:
Usually yes, if this file were also used by C/C++ code that defined these APIs. But since this header is meant for consumers of the static library who wish to re-export the defined functions (like what the simulator does) they need to be forward delcaed as `dllexport`. A consumer who is not re-exporting the APIs would be expected to forward declare them, similar to how they are handled in Rust or QIR itself.